README.md

```diff
@@ -2,7 +2,7 @@
 
 Adds fields to disclose the financing of the whole process and its individual contracts.
 
-Sometimes, particularly in the case of Public Private Partnerships, contracts are financed using a range of instruments, including loans, grants, share issues and so-on. This information can be updated over the lifetime of the contract.
+Sometimes, particularly in the case of Public Private Partnerships, contracts are financed using a range of instruments, including loans, grants, share issues and so-on. This information can be updated over the lifetime of the contract. In other cases, the whole contracting process is financed by some of these instruments.
```

Comment:
@jpmckinney I also added this sentence, but I can remove it if it is not necessary

Reply:
I've removed it, as the first paragraph already communicates that financing can be about the whole contracting process. This paragraph is just about explaining why contracts also have financing data.